src/platform/osx.m

```diff
@@ -164,9 +164,11 @@ int platform_open_common_file_dialog(int type, utf8 *title, utf8 *filename, utf8
 		panel.allowedFileTypes = extensions;
 		panel.directoryURL = [NSURL fileURLWithPath:directory];
 		if ([panel runModal] == NSFileHandlingPanelCancelButton){
+			SDL_RaiseWindow(gWindow);
```

Comment:
I was thinking it could go before the if-statement..

```diff
+SDL_RaiseWindow(gWindow);
 if ([panel runModal] == NSFileHandlingPanelCancelButton){
   return 0;
 } else {
   strcpy(filename, panel.URL.path.UTF8String);
   return 1;
 }
```